dockerfiles/theia/ubi8-nodejs10.Dockerfile

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) 2018 Red Hat, Inc.
```

Comment:
it seems it's a big copy/paste difficult to maintain file

I would opt it for a given stage inside the existing Dockerfile

and just use one given target against that docker build
docker build --target <ubi>